py4cast/cli.py

```diff
@@ -0,0 +1,140 @@
+from lightning.pytorch.cli import LightningCLI
+
+from py4cast.lightning import AutoRegressiveLightning, PlDataModule
+from lightning.pytorch.callbacks import BasePredictionWriter
+
+from py4cast.io.outputs import GribSavingSettings, save_named_tensors_to_grib
+
+
+class GribWriter(BasePredictionWriter):
```

Comment:
What is the rationale behind addind this clas in this file ?

Reply:
ok, I add it to a specific callbacks.py file. It could be useful to have a file dedicated to callbacks.